BackendTaskPilot/Program.cs

```diff
@@ -0,0 +1,38 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
```

Comment:
Should our API be public and therefore accessible to all or do we want to restrict its access and therefore implement authentication? This will depend of its use and we should have a chat with our project manager.